### PR TITLE
Allow to set default log cleanup parameters from Log Initialization

### PR DESCRIPTION
**Pull request by Averus-a**
### Description of Change ###

Purposes of change: we should be able to define how many and how big Log files we want to store

TODO:
- [ ] Add changelog

### Issues Resolved ### 

- fixes #

### API Changes ###

 
None

### Platforms Affected ### 

- All
- WPF
- UWP
- iOS
- Android

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] I have included examples or tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file
- [x] Rebased on top of the target branch at time of PR
- [x] Changes adhere to coding standard

### CODE REVIEW

File: src/Orchestra.Core/Helpers/LogHelper.cs [hunk 1]
```diff
@@ -47,6 +47,15 @@ public static class LogHelper
     {
         private static readonly ILog Log = LogManager.GetCurrentClassLogger();
 
+        /// <summary>
+        /// Maximum Log file size in KBs
+        /// </summary>
+        public static int MaxFileLogSizeDefault = 10 * 1024;
+
+        public static int MaxLogFileArchiveDaysDefault = 7;
```
Comment (GeertvanHorrik) [inline src/Orchestra.Core/Helpers/LogHelper.cs:55]: Please use 14

File: src/Orchestra.Core/Helpers/LogHelper.cs [hunk 2]
```diff
@@ -47,6 +47,15 @@ public static class LogHelper
...
+
+        public static int MaxLogFileArchiveFilesCountDefault = 10;
```
Comment (GeertvanHorrik) [inline src/Orchestra.Core/Helpers/LogHelper.cs:57]: Please use 20

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]
The defaults are added, but how can someone override? I think we need properties for this?

**Comment by Averus-a** [comment]
> The defaults are added, but how can someone override? I think we need properties for this?

image
The first option like this
Or we can provide configuration settings

**Review by GeertvanHorrik — APPROVED** [review]

**Pull request merged**